src/main/scala/mesosphere/marathon/core/volume/providers/Helpers.scala

```diff
@@ -53,7 +57,7 @@ protected[providers] object OptionSupport {
   }
 
   /** a validator to enforce that values parse to booleans */
-  lazy val booleanValidator: Validator[String] = new Validator[String] {
+  lazy val validBoolean: Validator[String] = new Validator[String] {
```

Comment:
could be written as `Validation.isTrue(s"Expected a valid boolean", Try(_.toBoolean).isSuccess)`